ext/opcache/jit/zend_jit.c

```diff
@@ -2369,6 +2364,11 @@ static int zend_jit(const zend_op_array *op_array, zend_ssa *ssa, const zend_op
 								}
 							}
 						}
+						if (opline->op2_type != IS_CONST
+						 || Z_TYPE_P(RT_CONSTANT(opline, opline->op2)) != IS_STRING
+						 || Z_STRVAL_P(RT_CONSTANT(opline, opline->op2))[0] == '\0') {
+							break;
+						}
```

Comment:
Why did you move this down?
I suppose, you relay on ``ce`` selected above in non-optimized path.
This leads to false positive build failures, because of possibly uninitialized ``ce``. This should be fixed.

I'm not completely sure if this code is going to be right for properties overridden with hooked ones in children.
Let we generate the code for parent class context where ``ce->num_hooked_props`` is zero. Then we execute it in context of child that overrides the accessed property...@iluuu1994 please check this

Reply:
> Why did you move this down? I suppose, you relay on `ce` selected above in non-optimized path. This leads to false positive build failures, because of possibly uninitialized `ce`. This should be fixed.

Indees that is the reason for moving down. I can fix build failure.

> I'm not completely sure if this code is going to be right for properties overridden with hooked ones in children. Let we generate the code for parent class context where `ce->num_hooked_props` is zero. Then we execute it in context of child that overrides the accessed property...@iluuu1994 please check this

I only skip rhe check for final classes, so this should not be a problem.

Reply:
> I only skip the check for final classes without hooks, so this should not be a problem.

Right. Then this shouldn't be a problem.